src/Rest/RestClient.php

```diff
@@ -89,6 +91,15 @@ class RestClient implements RestClientInterface {
 
   protected $httpClientOptions;
 
+  /**
+   * Token storage.
+   *
+   * @var \Drupal\salesforce\Storage\SalesforceAuthTokenStorage
+   *
+   * @deprecated interim, do not use.
```

Comment:
Change deprecated message to "Legacy Auth Scheme, do not use, will be removed." for all instances.